### ISSUE TITLE
Why is confidence interval so large for COD?

### ISSUE THREAD

**Issue by Kirkman** [erhla/cmfproperty#7]
Hi @erhla, I recently used cmfproperty to analyze sales data for St. Louis County, Mo., for the period 2010 to 2023.

Everything worked well, but I am puzzled by one aspect of the final report that was generated.

The diagram for the Coefficient of Dispersion in our report shows a very large confidence interval -- far bigger than the CI on the COD chart for St. Louis County on the [CMF's 2020 report online](https://s3.us-east-2.amazonaws.com/propertytaxdata.uchicago.edu/nationwide_reports/web/St.%20Louis%20County_Missouri.html). (So big that it's changing the Y-axis scale)

Here's a comparison:

| On CMF website | In our new report |
| ------------------ | ------------------- | 
| ![cmf](https://github.com/erhla/cmfproperty/assets/761387/6374401c-b6bc-46f5-8fe2-2d9a2ee34204) | ![pd](https://github.com/erhla/cmfproperty/assets/761387/99327763-431d-4faf-a382-58f878cf3af8) |

None of the other diagrams in our new report have this issue. All the other diagrams' confidence intervals are very tightly aligned to the data lines, similar to those in the 2020 CMF report.

Anyway, I'm happy to send the source sales data we used (it's a 17mb CSV), or to share my code.

**Comment by erhla** [comment]
Thanks Josh. Can you rerun the report and attach using diagnostic function? That's designed to try to answer issues like this.

`make_report(ratios, 'jursidiction_name', diagnostic=TRUE)`

**Comment by Kirkman** [comment]
I re-ran the report this morning with `diagnostic=True`, though nothing really stuck out to me among the diagnostic output, except maybe that the script is inflation-adjusting to 2021, but I have data through 2023.

```
            PARID TAXYR ASMTOT SALEYR  PRICE
           <char> <int>  <num>  <int>  <int>
     1: 03H110106  2010  48710   2010 143920
     2: 03H110106  2010  48710   2010 143920
     3: 03H120082  2010  45100   2010 205900
     4: 03H120082  2010  45100   2010 205900
     5: 03H120143  2010  74030   2010 460000
    ---
525547: 36J430180  2023  36650   2023  62500
525548: 36K640191  2023  68720   2023 425000
525549: 36K640191  2023  68720   2023 425000
525550: 36K640553  2023  83620   2023 445000
525551: 36K640553  2023  83620   2023 445000

[1] "Filtered out non-arm's length transactions"
[1] "Inflation adjusted to 2021"
R[write to console]:

processing file: report.Rmd

R[write to console]: output file: report.knit.md

"C:/PROGRA~3/CHOCOL~1/bin/pandoc" +RTS -K512m -RTS report.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output pandoc58867ca62c0.html --lua-filter "C:\Users\xxxxx\AppData\Local\R\win-library\4.3\rmarkdown\rmarkdown\lua\pagebreak.lua" --lua-filter "C:\Users\xxxxx\AppData\Local\R\win-library\4.3\rmarkdown\rmarkdown\lua\latex-div.lua" --embed-resources --standalone --variable bs3=TRUE --section-divs --table-of-contents --toc-depth 2 --variable toc_float=1 --variable toc_selectors=h1,h2 --variable toc_collapsed=1 --variable toc_smooth_scroll=1 --variable toc_print=1 --template "C:\Users\xxxxx\AppData\Local\R\win-library\4.3\rmarkdown\rmd\h\default.html" --no-highlight --variable highlightjs=1 --number-sections --variable theme=sandstone --mathjax --variable "mathjax-url=https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" --include-in-header "C:\Users\xxxxx\AppData\Local\Temp\Rtmps5CJ31\rmarkdown-str588fe741c8.html" --citeproc
R[write to console]:
Output created: S:\yyyyyy\assessment-data\analyzed-data\/St. Louis County, Missouri.html

[1] "~~~~~~~~~~~~"
[1] "geom warnings can be disregarded"
[1] "Report created at S:\\yyyyyy\\assessment-data\\analyzed-data\\/St. Louis County, Missouri.html"
```

Also, on Mar 22, I emailed you a copy of the sales data I'm using.
